src/test/java/lotto/LottoNumbersTest.java

```diff
@@ -11,8 +11,26 @@
 public class LottoNumbersTest {
 
     @Test
-    @DisplayName("로또 1장 생성 테스트")
-    public void lottoMake(){
-        assertThat(LottoNumber.getLottoNumbers()).hasSize(6);
+    @DisplayName("로또 캐싱 생성 객체 비교")
+    public void lottoObjectEqual(){
+        LottoNumber lottoNumber1 = LottoNumber.of(1);
+        LottoNumber lottoNumber2 = LottoNumber.of(1);
+        assertThat(lottoNumber1).isEqualTo(lottoNumber2);
```

Comment:
```suggestion
        assertThat(lottoNumber1 ==lottoNumber2 ).isTrue();
```
캐싱을 테스트하려면 `==`이 true인지를 검증해야 하지 않을까?